### DISCUSSION TITLE
Can't manage playback queue from CLI

### DISCUSSION THREAD

**Discussion by altAccount23** [DeaDBeeF-Player/deadbeef#3147] [General]
Is there an equivalent to the play next button in the CLI? Can i insert/append to the playback queue with a command somehow?

**Comment by Oleksiy-Yakovenko** [comment]
Yes, it is possible to do at least part of what you wrote. Maybe not all. Depends on specifics.
What problem did you encounter when you tried doing any of that?

**Reply by altAccount23** [reply]
The only option I can find in the --help menu that seems like it could do this is --queue but that only appends to a playlist. Sorry if I'm just stupid, thx for your time.

**Reply by Oleksiy-Yakovenko** [reply]
Well yeah.. playlist is sort of like a playback queue.
There's another playback queue which is usable with "play next" and "play later" menu items. which is like another playlist that takes precedence, but it's not accessible from command line.

**Reply by Oleksiy-Yakovenko** [reply]
There are already some feature requests about this: #1669, #602, #2483, and more. But controlling playback queue from command line is not possible.
